src/Integration.Vsix/SonarLintDaemon/Protocol.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.SonarLintDaemon
+{
+
+    class Request
```

Comment:
If this class is straight port of an existing Java class it might be worth calling this out in a comment and include a link to the Java class in GitHub.